src/aind_data_asset_indexer/utils.py

```diff
@@ -672,37 +673,6 @@ def copy_then_overwrite_core_json_files(
                 Key=target,
             )
             _log_message(message=response, log_flag=log_flag)
-            # Overwrite core fields from metadata.nd.json to the s3 core jsons
```

Comment:
moved this part out of copy util method, since identical to `sync_core_json_files`